NSS-Day-Cohort-42/trunch-bunch-truncheons-and-flagons
# Monica's Branch

# Steps To Review

1. `git fetch --all`
1. `git checkout mp`
1. make sure you're serving 
1. make sure the page renders the navbar and the nav links. make sure the list of scores shows up. 
1. view code to confirm Score.js, ScoreList.js, and ScoreProvider.js are error-free.
1. view code to confirm not extraneous errors exist.
